src/forms/ButtonReset.js

```
@@ -21,8 +21,9 @@ const ButtonReset = function ({ onClick, snippets, overrides }) {
 
   return (
     <BigButton
-      onClick = { clickHandler }
-      { ...overrides }>
+      className = { `caution` }
+      onClick   = { clickHandler }
+      overrides = { overrides }>
       { snippets.i_newClient }
```

It looks like `ButtonReset` doesn't use `props.children`; can we have this render `{ children }` instead of the snippet? Or if this will never be used with any other text, not pass children in `VisitPage`?

---

lol, that occurred to me late last night after I'd already made the changes to the `VisitPage` changes. In general, I kind of dislike having that kind of silent default. Obviously it's worse than a silent default right now, though. It'll have to change and I'll go ahead and do it for this PR.